From 600e11061e8c01950187561ffe8572583a84132d Mon Sep 17 00:00:00 2001
From: Randy Li <randy.li@rock-chips.com>
Date: Sat, 1 Apr 2017 10:55:07 +0800
Subject: [PATCH] ARM: dts: rockchip: enable 1000Mbps of rk3288

Because of the bad quality of EVB board, its phy will
fail at the GMAC mode. While I have verified the ASUS
tinker, it doesn't have such problem.

Change-Id: I3306b918363dff513e279762482f1ce1eaa30df8
Signed-off-by: Randy Li <randy.li@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb.dtsi | 1 +
 arch/arm/boot/dts/rk3288.dtsi     | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3288-evb.dtsi b/arch/arm/boot/dts/rk3288-evb.dtsi
index 1442b6179d57..45950680fc40 100644
--- a/arch/arm/boot/dts/rk3288-evb.dtsi
+++ b/arch/arm/boot/dts/rk3288-evb.dtsi
@@ -357,6 +357,7 @@
 	pinctrl-0 = <&rgmii_pins>;
 	tx_delay = <0x30>;
 	rx_delay = <0x10>;
+	max-speed = <100>;
 	status = "okay";
 };
 
diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index 9e8e11a3fb03..79e10bcf530d 100644
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -637,7 +637,6 @@
 			"aclk_mac", "pclk_mac";
 		resets = <&cru SRST_MAC>;
 		reset-names = "stmmaceth";
-		max-speed = <100>;
 		status = "disabled";
 	};
 
-- 
2.35.3

